### PR TITLE
Update maintenance page and static assets

### PR DESCRIPTION
**Pull request by edwardhorsford**
### Context

This updates the maintenance page and assets.

Updates:
* Using latest govuk-template markup
* Update to latest govuk-frontend static assets
* Footer and header match Register's
* Content updated, now including space for timestamped status updates
* Template messages included
* Updated instructions

Screenshot 2022-10-28 at 17 18 15

### Guidance to review

Would appreciate if a dev could try deploying this to check it looks good.

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

**Comment by edwardhorsford** [comment]
[Update Register maintenance page](https://trello.com/c/wrHRluer/4909-update-register-maintenance-page)

File: docs/maintenance-mode.md [hunk 1]
```diff
@@ -4,7 +4,14 @@ The repo includes a simple service unavailable page page that can be pushed to a
 
 When enabled, all requests will receive the [service unavailable page](/service_unavailable_page/web/public/internal/index.html).
 
-### Enable Maintenance mode
+## Instructions
```
Comment (AbigailMcP) [inline docs/maintenance-mode.md:7]: This is going to conflict with this i think https://github.com/DFE-Digital/register-trainee-teachers/pull/2847/files

File: service_unavailable_page/web/public/internal/index.html [hunk 2]
```diff
@@ -1,85 +1,150 @@
...
+
+              <h2 class="govuk-heading-s">
+                1:11pm, Monday 1 January 2022
+              </h2>
+              <p class="govuk-body">
+                We’re aware of an issue with Register. We’re working to identify and fix this to make the service available again.
+              </p>
+
+              <!--
+              Example updates:
```
Comment (AbigailMcP) [inline service_unavailable_page/web/public/internal/index.html:89]: Love these!

**Review by AbigailMcP — APPROVED** [review]

File: docs/maintenance-mode.md (hunk 1 above)
Comment (edwardhorsford) [reply docs/maintenance-mode.md:7]: Sucks to be @gpeng 🤣

**Pull request merged**